install-paperless-ngx.sh

```diff
@@ -318,7 +317,7 @@ DEFAULT_LANGUAGES="deu eng fra ita spa"
 	echo "PAPERLESS_TIME_ZONE=$TIME_ZONE"
 	echo "PAPERLESS_OCR_LANGUAGE=$OCR_LANGUAGE"
 	echo "PAPERLESS_SECRET_KEY=$SECRET_KEY"
-	if [[ ! " ${DEFAULT_LANGUAGES[@]} " =~ " ${OCR_LANGUAGE} " ]] ; then
+	if [[ ! " ${DEFAULT_LANGUAGES[*]} " =~ ${OCR_LANGUAGE} ]] ; then
```

Comment:
Same here

Reply:
I tested this one out as well.  When I set `it` as the OCR_LANGUAGE, PAPERLESS_OCR_LANGUAGE was set to `it` instead of `eng`.  That looks to be working.